extension/test/unit/submit-payment-details.handler.spec.js

```diff
@@ -0,0 +1,144 @@
+    }
+  }
+  /* eslint-enable max-len */
+
+  beforeEach(() => {
+    scope = nock(`${config.adyen.apiBaseUrl}`)
+  })
+
+  afterEach(() => {
+    nock.cleanAll()
+  })
+
+  it('when resultCode from Adyen is "Authorized", '
+    + 'then it should return actions "addTransaction", "addInterfaceInteraction" and "setCustomField"', async () => {
+    scope.post('/payments/details')
+      .reply(200, submitPaymentDetailsSuccessResponse)
+
+    const ctpPaymentClone = _.cloneDeep(ctpPayment)
+    ctpPaymentClone.custom.fields.submitAdditionalPaymentDetailsRequest = JSON.stringify(submitPaymentDetailsRequest)
+    ctpPaymentClone.custom.fields.makePaymentResponse = JSON.stringify(makePaymentRedirectResponse)
+
+    const response = await execute(ctpPaymentClone)
+    console.log(`response ${JSON.stringify(response.actions)}`)
+
+    expect(response.actions).to.have.lengthOf(3)
+    const addInterfaceInteraction = response.actions.find(a => a.action === 'addInterfaceInteraction')
+    expect(addInterfaceInteraction.fields.type).to.equal('submitPaymentDetails')
```

Comment:
it could be nice to use constants and manage the string in only 1 place, same for all constant references.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/commit/60b3fce16eca6b0ca138ff33dc7fce6371d7bdec